_pkgdown.yml

```diff
@@ -16,6 +16,23 @@ navbar:
           href: StandardReportSite.html
         - text: Country Report
           href: StandardReportCountry.html
+    # articles:
```

Comment:
we want to include these again eventually, right? i just commented them out for the time being